TTS/api.py

```diff
@@ -25,39 +25,29 @@ def __init__(
         progress_bar: bool = True,
         gpu=False,
     ):
-        """🐸TTS python interface that allows to load and use the released models.
+        """Initialize the TTS module.
 
         Example with a multi-speaker model:
             >>> from TTS.api import TTS
-            >>> tts = TTS(TTS.list_models()[0])
-            >>> wav = tts.tts("This is a test! This is also a test!!", speaker=tts.speakers[0], language=tts.languages[0])
+            >>> tts = TTS(model_name=TTS().list_models().list_models()[0], gpu=gpu)
```

Comment:
I think that the GPU  parameter should be True or False here because in this way the user can easily understand the parameter.